File: proy/conexion/demo/src/main/java/uniandes/edu/co/proyecto/scripts/insercion/producto.js
db.producto.insertOne({
  id: 1,
  nombre: "Leche Entera",
  costo_bodega: 1500,
  precio_unidad: 2000,
  presentacion: "Paquete de 6 unidades",
  cantidad_presentacion: 6,
  volumen: 1000,
  peso_empaque: 150,
  unidad_medida: "ml",
  codigo_barras: "123456789",
  fecha_expiracion: ISODate("2025-01-01T00:00:00Z"),
  categoria: {
    codigo: 101,
    nombre: "Perecederos",
    descripcion: "Productos que requieren refrigeración",
    caracteristica_almacenamiento: "Mantener entre 0-4 °C",
  },
});

File: MiniParranderosNoSQL-ConexionIni/repo/demo/src/main/java/uniandes/edu/co/proyecto/modelo/Producto.java
package uniandes.edu.co.proyecto.modelo;

import java.util.Date;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.ToString;

@Document(collection = "producto")
@ToString
public class Producto {
    @Id
    private int id;
    private String nombre;
    private int costo_bodega;
    private int precio_unidad;
    private String presentacion;
    private int cantidad_presentacion;
    private double volumen;
    private double peso_empaque;
    private String unidad_medida;
    private String codigo_barras;
    private Date fecha_expiracion;
    private Categoria categoria;

}
